boards/shields/mcp2518fd_click/mcp2518fd_click.overlay

```diff
@@ -0,0 +1,28 @@
+&mikrobus_spi {
+	cs-gpios = <&mikrobus_header 2 GPIO_ACTIVE_LOW>;
+	mcp2518fd: canfd@0 {
```

Comment:
See https://docs.zephyrproject.org/latest/hardware/porting/shields.html#shield-porting-and-configuration:
> Besides, in order to avoid name conflicts with devices that may be defined at board level, it is advised, specifically for shields devicetree descriptions, to provide a device nodelabel is the form <device>_<shield>, ....